### DISCUSSION TITLE
[SOLVED] Using with Redux-Saga

### DISCUSSION THREAD

**Discussion by Jlexyc** [invertase/react-native-firebase#5073] [Q&A]
UPD. Issue solved, thanks!

I'm trying to use react-native-firebase functions in Redux-Saga. But it always losing its context (seems like) when it's called within effect "call".

`const auth = firebase.auth()`
`yield call([auth, auth.signInAnonymously])`

And I'm getting:
`TypeError: E is not a function. (In 'E()', 'E' is undefined)`

Tried to bind context, but the result the same.
Any suggestions?

**Comment by davidepalazzo** [comment]
Hi @Jlexyc, have a look at [this article](https://medium.com/@astrosuf/firebase-with-redux-saga-3b8d509ce522). It should hopefully  fix your problem.

**Reply by Jlexyc** [reply]
Thanks for the reply, but it doesn't help as I'm doing exactly the same :(. Maybe the author used the previous version of react-native-firebase or I'm missing something.

**Reply by Jlexyc** [reply]
@davidepalazzo 
Thank you for your attempt to help. I've found an issue, it was my mistake :)

**Reply by davidepalazzo** [reply]
Great!
